kibble/render/language_renderer.go

```diff
@@ -0,0 +1,72 @@
+//    Copyright 2018 SHIFT72
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package render
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"kibble/models"
+)
+
+// Setup and Create language files based on API or local language files.
+func WriteLanguageFiles(site *models.Site, sourcePath string) error {
```

Comment:
Suggest tweaking the method name because it doesn't always write files.

```suggestion
// Ensure the language files are correct for the specified source, either the API or local files.
func EnsureLanguageFiles(site *models.Site, sourcePath string) error {
```

Reply:
I was thinking about moving the translation API check out of the function, and place the call inside the render API functions check which was also cut down on another` site.Toggles["translations_api"]`